### PR TITLE
GAMETRADER-77

### PR DESCRIPTION
**Pull request by Krystiano99**

### CODE REVIEW

File: src/components/footer/Footer.module.scss [hunk 1]
```diff
@@ -1,38 +1,109 @@
...
   background-color: var(--gt-secondary);
+  display: flex;
+  align-items: center;
 }
 
-.container{
+.footer{
   height: 100%;
-  width: 100%;
+  //width: 100%;
```
Comment (vroniu) [inline src/components/footer/Footer.module.scss]: this line can be deleted

File: src/components/footer/Footer.module.scss [hunk 2]
```diff
@@ -1,38 +1,109 @@
...
 }
 
-.leftSection{
-  width: 25%;
+
+.footer .centerSection{
+  display: flex;
+  justify-content: center;
+  order: 1;
 }
```
Comment (vroniu) [inline src/components/footer/Footer.module.scss]: use Sass nested selectors here

File: src/components/footer/Footer.module.scss [hunk 3]
```diff
@@ -1,38 +1,109 @@
...
 
-.centerSection{
-  width:100%;
-  text-align: center;
+.footer .centerSection p{
+  margin: 0;
 }
```
Comment (vroniu) [inline src/components/footer/Footer.module.scss]: use Sass nested selectors here

File: src/components/footer/Footer.module.scss [hunk 4]
```diff
@@ -1,38 +1,109 @@
...
 
-.centerSection p{
-  margin-bottom: 0;
+.footer button{
+  width: 100%;
+  margin-bottom: 1rem;
 }
```
Comment (vroniu) [inline src/components/footer/Footer.module.scss:30]: use Sass nested selectors here

File: src/components/footer/Footer.module.scss [hunk 5]
```diff
@@ -1,38 +1,109 @@
...
-  display: flex;
-  justify-content: right;
+
+@media (min-width: 600px) {
+  .footer{
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+  }
+  .footer button{
```
Comment (vroniu) [inline src/components/footer/Footer.module.scss]: use Sass nested selectors here and below

File: src/components/footer/Footer.module.scss [hunk 6]
```diff
@@ -1,38 +1,109 @@
...
-.rightSection button{
-  margin: 0 0 0 10px;
+
+
+@media (min-width: 960px) {
+
+  .footer .leftSection{
+    justify-content: left;
+  }
+  .footer .xd{
```
Comment (vroniu) [inline src/components/footer/Footer.module.scss]: .xd isnt used anywhere in this app, this selector can be deleted

File: src/components/footer/Footer.module.scss [hunk 7]
```diff
@@ -1,38 +1,109 @@
...
+@media (min-width: 1440px) {
+  .footer button{
+    width: auto;
+  }
+  .footer .leftSection{
+    justify-content: left;
+  }
+
+  .footer .xd .rightSection button{
+
```
Comment (vroniu) [inline src/components/footer/Footer.module.scss]: dont leave empty css selectors

File: src/components/footer/Footer.module.scss [hunk 8]
```diff
@@ -1,38 +1,109 @@
...
+@media (min-width: 1920px) {
+  .footer button{
+    width: auto;
+  }
+  .footer .leftSection{
+    justify-content: left;
+  }
+
+  .footer .xd .rightSection button{
+
```
Comment (vroniu) [inline src/components/footer/Footer.module.scss:96]: dont leave empty css selectors

**Review by vroniu — CHANGES_REQUESTED** [review]
App looks good, code needs a little bit of cleaning - mainly removing unnecessary CSS classes and using Sass nested selectors.

File: src/components/footer/Footer.module.scss (hunk 6 above)
Comment (Krystiano99) [reply src/components/footer/Footer.module.scss]: xd

**Pull request merged**